gitbook-docs/data_model.md

```diff
@@ -171,9 +171,10 @@ A delta message can be recognised from the other types by the topmost level havi
 `updates` is the only required property.
 If `context` is missing it is assumed that the data is related to the `self` context.
 
-Context is a path from the root of the full tree. In this case 'vessels.urn:mrn:imo:mmsi:234567890'. All subsequent data is relative to
-that location. The context could be much more specific, e.g. 'vessels.urn:mrn:imo:mmsi:234567890.navigation', whatever is the common root
-of the updated data.
+Context is a path from the root of the full tree to the _container object_, which is always a vessel with the current Signal K version.
+The delimiter in the context path is `.` (period).
```

Comment:
But we are consistent: paths in property names and values like in full tree `$source` and `values` property keys use period delimiter. If the `values` property keys used / as delimiter and you wanted to retrieve a certain value via REST / would need escaping.

Not saying everything is perfect, but change to / would not be backwards compatible, as would the structured context you suggest.